packages/clerk-expo-passkeys/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@clerk/clerk-expo-passkeys",
```

Comment:
Should we go with `@clerk/expo-passkeys` instead?

Reply:
I don't have a strong preference for the naming, the reason I used this one is to match the pattern that we currently have for `@clerk/clerk-expo`

Reply:
Renamed to `@clerk/clerk-expo`